<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserSelection</class>
 <widget class="QWidget" name="UserSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Selection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>RaDoTech Health Monitor</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="batteryLabel">
       <property name="text">
        <string>Battery: 100%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="selectUserLabel">
     <property name="text">
      <string>Select User Profile:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="userListWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="addUserButton">
       <property name="text">
        <string>Add New User</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteUserButton">
       <property name="text">
        <string>Delete User</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectUserButton">
       <property name="text">
        <string>Select User</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>